test/node/conversation-labels.js

```diff
@@ -14,6 +14,8 @@ describe('Labels', () => {
     before(async () => {
         client = new Circuit.Client(config.bot1);
         await client.logon();
+        const topic = `${Date.now()}a`;
+        conversation = await client.createConferenceBridge(topic);
```

Comment:
Could you use the preparation conversation here?

Reply:
done